ledger/db/src/test_utils/mock_ledger.rs

```diff
@@ -300,11 +301,139 @@ pub fn get_test_ledger_blocks(n_blocks: usize) -> Vec<(Block, BlockContents)> {
     blocks_and_contents
 }
 
+/// Get blocks with custom content in order to simulate conditions seen in
+/// production
+///
+/// * `outputs_per_recipient_per_block` - transaction outputs per account per
+///   block
+/// * `num_accounts` - number of accounts in the blocks
+/// * `num_blocks` - number of simulated blocks to create
+/// * `key_images_per_block` - number of key images per block
+/// * `max_token_id` - number of distinct token ids in blocks
```

Comment:
The doc for `max_token_id`  says "number of distinct token ids", however the computation for `num_outputs` adds `1` and the loop below using `max_token_id` is inclusive.
This parameter is actually one less than the number of distinct token ids.

```suggestion
/// * `max_token_id` - maximum distinct token ids in blocks. This is inclusive 
///     i.e. if this is `2` then the ids in the block will be (0, 1, 2)
```
I don't like how verbose the argument doc is in the suggestion, just not sure it's intuitive that `max_token_id` is inclusive from the current docs alone.